docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -0,0 +1,75 @@
+
+.. figure:: ../../_assets/RepresentingChainsBODS-RealWorld.svg
+   :alt: Person 1 indirectly holds a 15 percent shareholding in Company E, via an intermediary: Company A. Person 2 directly holds 32 precent of Company E's shares.
+   :width: 500px
+   :align: center
+
+In the above situation, Person 1 *indirectly* benefits from a 15% shareholding in Company E and Person 2 has a *direct* 32% shareholding in Company E. In a jurisdiction where people with a shareholding (direct or indirect) of over 10% in a company should be disclosed as beneficial owners, both Person 1 and Person 2’s interests would be declared. Additionally, the jurisdiction may require that details of Person 1’s indirect interest are disclosed. That is: that some details of the chain Company E - Company A - Person 1 are also disclosed. 
+
+In BODS, the following properties are used to represent such information disclosure:
+
+* ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
+* ``directOrIndirect`` (See :ref:`schema-interest`)
+* ``componentStatementIDs`` (See :ref:`schema-ownership-or-control-statement`)
+* ``isComponent`` (See :ref:`schema-entity-statement`)
+
+Requirements
+------------------------
+
+Representing beneficial ownership
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a person is a beneficial owner of an entity, E, whether directly or indirectly, and one of them is required to declare this beneficial ownership:
+
+1. There MUST be a *primary* Ownership-or-control Statement connecting the two which represents the beneficial ownership relationship. In particular: 
+
+   a. the entity’s Statement MUST be the ``subject``;
+   b. the person’s Statement MUST be the ``interestedParty``;
+   c. ``isComponent`` MUST be false;
+   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included in this primary Ownership-or-Control Statement if known; and
+   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'.
+
+2. If beneficial ownership is known to be exercised indirectly, via intermediary entities or arrangements then the ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
+
+Representing intermediaries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Where beneficial ownership is known to be exercised indirectly, via known intermediary entities or arrangements, this SHOULD be represented in addition to the above. In particular:
+
+1. The chain of known intermediary entities and arrangements SHOULD be represented by *secondary* Entity Statements, Ownership-or-control Statements and Person Statements.
+2. These secondary statements SHOULD link the beneficial owner’s Statement to entity E’s statement indirectly.
```

Comment:
I know that we reference E in the preamble to this section, but all of the numbered lists use general terms apart from here. Is that because we don't have an agreed term for the "target entity", ie the one at the bottom of the chain of ownership?

I think that

"These secondary statements SHOULD link the beneficial owner’s Statement to the target entity (Company E in our example)’s statement indirectly."

is preferable, although it could do with some rephrasing so as not to have an apostrophe after the bracket.

Reply:
> Is that because we don't have an agreed term for the "target entity"

Yes - precisely. (And I've taken pains to write this so that it's equally applicable to person-centric declarations, so 'declaring entity' is also not an option.)

I don't really want to introduce a new term 'target entity' or to confuse this guidance (which is for the general case) by referring to 'Company E in our example' (which is a specific case).

... which makes me realise that I have confused the specific with the general by also using 'Company E' in the diagram and examples.

I've tried to unpick this with some edits.

Reply:
This reads really well now. I've just noticed that the subheading is the same as the title: "Representing beneficial ownership" 

It might be better to rephrase this and the following subheading slightly.

Reply:
Fixed